.gitignore

```diff
@@ -11,3 +11,4 @@
 /captures
 gradle
 .externalNativeBuild
+.cxx
```

Comment:
+ .cfx

Reply:
One year ago, you request `.cfx` ?

You could make an own pull request for this